uniffi_macros/src/util.rs

```diff
@@ -0,0 +1,64 @@
+use proc_macro2::Span;
+
+#[cfg(not(feature = "nightly"))]
+pub fn mod_path() -> syn::Result<Vec<String>> {
+    // Without the nightly feature and TokenStream::expand_expr, just return the crate name
```

Comment:
There are theoretically ways, but they're super hacky. This is not really much of a limitation. AFAIK, you already had to export all the symbols referenced in UDL from the crate root? That would be the central / only limitation for the non-nightly solution here.

Reply:
So we're talking Rust Nightly here? Can we add the Rust tracking issue in a comment then?

Reply:
Yes, Rust Nightly. I'll add a link to the tracking issue.

Also, I thought about the "re-export from crate root" part again – this isn't actually true, the exported function can be private and not accessible from the crate root. What matters is that the generated scaffolding function is a public symbol in the resulting cdylib. I can look into what is required for that. It might be that this nightly feature doesn't actually offer anything of value yet 😄 (what it could do later is provide bindings generators knowledge about the Rust module structure, but right now I don't think the bindings generators have the ability to output nested modules themselves, right?)